Source/Resource/Text/DataBase/Format.csv

```diff
@@ -125,9 +125,11 @@ Wave;;;A;Riff;;act at9 wav;audio/vnd.wave;;
 Wave64;;;A;Riff;;w64;;;
 WavPack;;;A;Wvpk;;wv wvc;;http://www.wavpack.com;
 Arri Raw;;;I;ArriRaw;;ari;;;
+ARW;;;I;Tiff;Sony Alpha RAW;arw;image/x-sony-arw;http://www.sony.com/;

```

Comment:
They are names with company name in the other PR about TIFF compression formats, so for coherency it should be there too, I think.